cobalt/build/get_build_id.py

```diff
@@ -14,21 +14,11 @@
 # limitations under the License.
 """Prints out the Cobalt Build ID."""
 
-import os
-from cobalt.build.build_number import GetOrGenerateNewBuildNumber
+from subprocess import check_call
 
 
 def main():
-  # Note $BUILD_ID_SERVER_URL will always be set in CI.
-  build_id_server_url = os.environ.get('BUILD_ID_SERVER_URL')
-  if build_id_server_url:
-    build_num = GetOrGenerateNewBuildNumber(version_server=build_id_server_url)
-    if build_num == 0:
-      raise ValueError('The build number received was zero.')
-    print(build_num)
-  else:
-    # No need to generate a build id for local builds.
-    print('0')
+  check_call(['git', 'rev-list', '--count', 'HEAD'])
```

Comment:
Should we offset this right away to distinguish builds off of the new counter ? Our current counting is ~300k something, maybe add a million ?